### PR TITLE
TIER3 Execution

### PR DESCRIPTION
**Pull request by mguetta1**

### CODE REVIEW

**Review by aufi — APPROVED** [review]
I'd suggest avoid wording "sensitive data" in tests, rather something like not public or tests that need credentials to access some repository or external service. Otherwise LGTM.

File: README.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ More advanced features like complex application analysis or some edge cases. Thi
 $ make test-tier2
 ```
 
+### Tier 3
+Private cases of Tier 1. This tier executes tests which include sensetive data (test configuration)
```
Comment (sshveta) [inline README.md:101]: We can change this to : 

This tier tests require credentials for some apps and will only be run through jenkins .These should be excluded from PR runs .

**Review by sshveta — APPROVED** [review]
